runtime-transaction/src/transaction_meta.rs

```diff
@@ -0,0 +1,26 @@
+//! Transaction Meta contains data that follows a transaction through the
+//! execution pipeline in runtime. Metadata can include limits specified by
+//! compute-budget instructions, simple-vote flag, transactino costs,
+//! durable nonce account etc; Metadata can be lazily populated as
+//! transaction goes through execution path.
+//!
+
+use {
+    solana_sdk::hash::Hash,
+};
+
+#[derive(Clone, Debug, Default, PartialEq, Eq)]
+pub struct TransactionMeta {
+    pub message_hash: Hash,
+    pub is_simple_vote_tx: bool,
```

Comment:
these feel like they should be `Option<T>` so we know if they've been populated or not